Source/Diagnostics/ReducedDiags/MultiReducedDiags.cpp

```diff
@@ -81,6 +82,11 @@ MultiReducedDiags::MultiReducedDiags ()
             m_multi_rd[i_rd]=
                 std::make_unique<ParticleNumber>(m_rd_names[i_rd]);
         }
+        else if (rd_type.compare("ParticleExtrema") == 0)
+        {
+            m_multi_rd[i_rd].reset
+                ( new ParticleExtrema(m_rd_names[i_rd]));
```

Comment:
Just for consistency with the ones around it and modernization:
```suggestion
            m_multi_rd[i_rd] = 
                std::make_unique< ParticleExtrema >(m_rd_names[i_rd]);
```